crates/bevy_window/src/window.rs

```diff
@@ -106,6 +106,8 @@ pub struct Window {
     pub resolution: WindowResolution,
     /// Stores the title of the window.
     pub title: String,
+    /// Stores the application ID of the window.
+    pub app_id: String,
```

Comment:
this is a field of a component, what happens if the user changes it after the window is created?

Reply:
Since there are no methods for modifying `app_id` after the app launches on [`winit`'s side](https://docs.rs/winit/latest/winit/window/struct.Window.html), I think we could make this a no-op and I have added some notes on this field about changes after the window is created.